src/zkevm_specs/evm/execution/error_Invalid_jump.py

```diff
@@ -0,0 +1,53 @@
+from ...util import FQ
+from ..instruction import Instruction, Transition
+from ..table import CallContextFieldTag
+from ..execution_state import ExecutionState
+from ..opcode import Opcode
+from ...util import N_BYTES_PROGRAM_COUNTER
+
+
+def invalid_jump(instruction: Instruction):
+    opcode = instruction.opcode_lookup(True)
+    # current executing op code must be JUMP or JUMPI
+    instruction.constrain_in(opcode, [FQ(Opcode.JUMP), FQ(Opcode.JUMPI)])
+    code_length = instruction.bytecode_length(instruction.curr.code_hash)
+    dest = instruction.stack_pop()
+    # lookup value from bytecode table
+    dest_value = instruction.rlc_to_fq(dest, N_BYTES_PROGRAM_COUNTER)
+
+    out_of_range, _ = instruction.compare(code_length, dest_value, N_BYTES_PROGRAM_COUNTER)
```

Comment:
`instruction.compare` checks `code_length < dest_value)`, but in fact it's also out of range when `code_length == dest_value`.

Reply:
changed to 
`    within_range, _ = instruction.compare(dest_value, code_length, N_BYTES_PROGRAM_COUNTER)`

  in  [d2eb296](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/296/commits/d2eb2961e90e2c5331a1346c09f9ed58914394a9). thanks